browser/help/faq/general/what-features-are-not-yet-in-v4-and-where-can-i-find-them.md

```diff
@@ -18,4 +18,3 @@ Below is a list of all features not included in the v4 MVP and where to find the
 | Regional Missense Constraint                    | Now available on v2 gene page                         |
 | Linkage disequilibrium scores                   | [v2](/downloads/#v2-linkage-disequilibrium) downloads |
 | Short Tandem Repeats (STRs)                     | [v3](/short-tandem-repeats?dataset=gnomad_r3)         |
```

Comment:
Should STRs also be removed too?

They were added here: https://github.com/broadinstitute/gnomad-browser/pull/1305